Content.Server/EntityEffects/Effects/ChemEyeDamageChange.cs

```diff
@@ -2,23 +2,25 @@
 using Content.Shared.Eye.Blinding.Systems;
 using JetBrains.Annotations;
 using Robust.Shared.Prototypes;
+using System.Text.Json.Serialization;
 
 namespace Content.Server.EntityEffects.Effects;
 
 /// <summary>
 /// Heal or apply eye damage
 /// </summary>
 [UsedImplicitly]
-public sealed partial class ChemHealEyeDamage : EntityEffect
+public sealed partial class ChemEyeDamageChange : EntityEffect
```

Comment:
```suggestion
public sealed partial class ChemChangeEyeDamage : EntityEffect
```
This better follows the naming convention. Same for the file name.